modules/rosa-getting-started-revoke-admin-privileges.adoc

```diff
@@ -20,7 +20,7 @@ ifdef::getting-started[]
 .Prerequisites
 
 * You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
-* You logged in to your Red Hat account by using the `rosa` CLI.
+* You logged in to your Red Hat account by using the {product-title} (ROSA) CLI (`rosa`).
```

Comment:
Same fixes suggested for https://github.com/openshift/openshift-docs/pull/61151/files#r1233482587